app/helpers/serializer_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# For serializer helpers
+module SerializerHelper
+  def team_id_from_app_id(app_id)
+    return if app_id.nil?
+
+    App.find_by(id: app_id)&.team_id
```

Comment:
Can you just do App.find(app_id) here instead of using .find_by?

Reply:
I decided to use `find_by` so it will return `nil` if it doesn't exist instead of throwing `ActiveRecord::RecordNotFound` and having to catch it